.travis.yml

```diff
@@ -54,17 +54,10 @@ script:
   - sudo sh -c "export DISPLAY=:1; cd securedrop/ && pytest -v tests/"
   - SECUREDROP_TESTINFRA_TARGET_HOST=travis testinfra -v testinfra/development/
   - pip freeze -l
-  # Docs linting. Performing this step *after* build VM tests pass, so as not
+  # Performing new pip install step *after* build VM tests pass, so as not
   # to alter the pip requirements, which would cause config tests to fail.
+  # The develop requirements include sphinx tooling for docs linting.
   - pip install -r securedrop/requirements/develop-requirements.txt
-  - make docs-lint
-  # flake8 linting
-  - >
-      flake8 testinfra securedrop-admin securedrop/*.py securedrop/management
-      securedrop/tests/functional securedrop/tests/*.py
-  # HTML linting
-  - >
-      html_lint.py --printfilename --disable=optional_tag,extra_whitespace,indentation
-      securedrop/source_templates/*.html securedrop/journalist_templates/*.html
+  - make lint
```

Comment:
I see why you did this, but it's probably better to run each linter separately - the reason being, if there's a failure on e.g. `make flake8`, then `make html-lint` will not run.